woocommerce/woocommerce-android
# Automate builds

This PR adds Fastlane lanes to help build the beta and final release versions of the app with the proper checks. 

# To Test
1. Run `bundle exec fastlane build_test_releases`.
2. Verify that the beta release has been build into the `artifacts` folder.
3. Install the .apk and the .aab (using `./tools/install-app-bundle.sh`) and verify that they works.
4. Run `bundle exec fastlane build_release`.
5. Verify that the system exists with an error because of the version name. 
6. Checkout a new branch, edit `build.gradle` changing the name to `2.2`, commit the changes.
7. Run `bundle exec fastlane build_release`.
8. Verify that the beta release has been build into the `artifacts` folder.
9. Install the .apk and the .aab (using `./tools/install-app-bundle.sh`) and verify that they works.


Update release notes:
